rma/models/rma.py

```diff
@@ -202,11 +204,15 @@ def _domain_location_id(self):
         compute="_compute_warehouse_id",
         store=True,
     )
-    reception_move_id = fields.Many2one(
-        comodel_name="stock.move",
+    reception_move_ids = fields.Many2many(
```

Comment:
Why are you changing this?

Reply:
Because i'm changing the create of the reception_move, it is now created with procurement run.
This is not returning a move. Thats why i'm doing this https://github.com/mt-software-de/rma/blob/00c8faaae4c4fb1bda0dafd56a964fd72680d6df/rma/models/rma.py#L698, https://github.com/mt-software-de/rma/blob/00c8faaae4c4fb1bda0dafd56a964fd72680d6df/rma/models/stock_move.py#L25 and https://github.com/mt-software-de/rma/blob/00c8faaae4c4fb1bda0dafd56a964fd72680d6df/rma/models/stock_move.py#L110